Sources/Quick/QuickSelectedTestSuiteBuilder.swift

```diff
@@ -0,0 +1,69 @@
+/**
+ Responsible for building a "Selected tests" suite. This corresponds to a single
+ spec, and all its examples.
+ */
+internal class QuickSelectedTestSuiteBuilder: QuickTestSuiteBuilder {
+
+    /**
+     The test spec class to run.
+     */
+    let testCaseClass: AnyClass!
+
+    /**
+     For Objective-C classes, returns the class name. For Swift classes without,
+     an explicit Objective-C name, returns a module-namespaced class name
+     (e.g., "FooTests.FooSpec").
+     */
+    var namespacedClassName: String {
```

Comment:
At first glance this reminded me of the [`-[XCTestCase languageAgnosticTestMethodName]`](https://github.com/modocache/XCTest-Public-Headers/blob/4b638808d0a742f03d4613f71e2eb03f3118498d/Developer/Platforms/iPhoneOS.platform/Developer/Library/Frameworks/XCTest.framework/XCTest/XCTestCase.h#L66) I came across back in the day. It seemed like a distracting point to bring up in my first review of this pull request, but now that you mention changing this method name... :wink: 

I think `className` is less descriptive. How would you feel about `moduleNamespacedClassName`, or something like that? I know it's long-winded, but I think it's pretty clear.